content/id/docs/templates/feature-state-deprecated.txt

```diff
@@ -0,0 +1,2 @@
+
+Fitur ini *deprecated*. Untuk informasi lebih lanjut mengenai tingkatan ini, silahkan merujuk pada [Kubernetes Deprecation Policy](/docs/reference/deprecation-policy/)
```

Comment:
We have to standardize `deprecated`

Between:
terdeprekasi / deprekasi / sudah usang / *deprecated*